code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1136,10 +1136,12 @@
 	sanity.setLevel(sanity.max_level)
 	restore_blood()
 //OCCULUS EDIT - Resets bodymarkings when we rejuv
+	if(!client)
+		..()
+		return
```

Comment:
and this block should prolly go under the `for(var/N in organs_by_name)` block

Reply:
Then we'd just end up cutting markings from mobs that most likely don't have any.

Or if our mob is a player mob without a ghost then we cut their markings without adding them back.

I'm pretty sure markings aren't interacted with in any way outside of character creation.

Reply:
ah, I didn't see that L1142-1144 was removing markings, I thought it was replacing organs. Whoops.